src/components/NewComment.jsx

```diff
@@ -0,0 +1,31 @@
+import { postComment } from "../utils/api"
+import { useState } from "react"
+
+export const NewComment = ({article_id}) => {
+    const [postedComment, setPostedComment] = useState()
+
+    const postNewComment = (e) => {
+        e.preventDefault()
+        setPostedComment({body: e.target[0].value, author: 'tickle122'})
+        postComment(article_id, e.target[0].value, 'tickle122')
+        .catch(() => {
+            setPostedComment('error')
+        })
+        e.target[0].value = ''
+    }
+
+    return <div><form onSubmit={(event) => {
+        postNewComment(event);
+      }} className="new-comment">
+        <input className="new-comment-body" type="text" name="body"></input>
```

Comment:
This needs to be textarea - to allow multi line comments. 
Also, its not currently a controlled component - react isnt in control of this at the moment and wouldnt know whats in the form until its submitted. You can run into rendering issues.